test/integration/che/cheAPI.go

```diff
@@ -0,0 +1,565 @@
+// +build integration
+
+/*
+Copyright (C) 2017 Red Hat, Inc.
```

Comment:
Why it is 2017? Isn't it should be 2018 and same in other files :smile:

Reply:
@budhram Originally created in 2017, I am just copy pasting it here. Was thinking about how to change the date and then kept it as it is... would `2017-2018` work?

Reply:
@budhram Looked on other files in Minishift and we just keep the oldest year, so I am keeping the 2017 here. This is a migration of code made in 2017 from minishift-addons repository, so imho the year should be original = 2017.

Reply:
I think 2017 is fine here if it was created in 2017 originally.